client/screens/user_profile/user_dropdown/user_dropdown.cpp

```diff
@@ -0,0 +1,22 @@
+#include "user_dropdown.h"
+#include "ui_user_dropdown.h"
+
+#include "QGraphicsDropShadowEffect"
+
+userDropdown::userDropdown(QWidget *parent) : QWidget(parent), ui(new Ui::userDropdown)
+{
+    ui->setupUi(this);
+
+    auto *effect = new QGraphicsDropShadowEffect;
```

Comment:
Memory leak - not deleting "effect" var, u allocated that on heap, bro.

Reply:
added parent argument, it'll deleted with dropdown